src/main/java/org/jnode/fs/xfs/inode/INodeBTreeRecord.java

```diff
@@ -3,6 +3,9 @@
 import org.jnode.fs.xfs.XfsObject;
 import org.jnode.fs.xfs.btree.BTreeRecord;
 
+import java.util.Arrays;
```

Comment:
Should be able to remove these unused imports

Reply:
The suggested change was applied, thanks